threewide/src/components/Tetris.tsx

```diff
@@ -573,6 +573,18 @@ const Tetris = ({
       ] = currentPiece.pieceType;
     }
 
+    console.log(placePieceLocation)
+    if(placePieceLocation[1] == 0){
```

Comment:
This logic is incorrect. The actual logic that game uses is that it checks all tile locations. If all of them don't fit in the 10x20 board then it's game over. Make sure you're getting the tile locations instead of the piece location.

Reply:
Also this makes it possible to place pieces above the 10x20 board so the board needs to be increased to 10x23

Reply:
Fixed myself